browser-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.3.11",
+  "dependencies": {
+    "@theia/core": "0.3.11",
+    "@theia/editor": "0.3.11",
+    "@theia/filesystem": "0.3.11",
+    "@theia/languages": "0.3.11",
+    "@theia/markers": "0.3.11",
+    "@theia/messages": "0.3.11",
+    "@theia/monaco": "0.3.11",
+    "@theia/navigator": "0.3.11",
+    "@theia/preferences": "0.3.11",
+    "@theia/process": "0.3.11",
+    "@theia/terminal": "0.3.11",
+    "@theia/typescript": "0.3.11",
+    "@theia/workspace": "0.3.11",
+    "@eclipse/che-theia-factory": "^0.3.11"
```

Comment:
carret should be removed as well
also I'm not sure this extension should use 0.3.11  version , it's the first version so more like 0.0.1 (or something like that)